kolibri/plugins/learn/assets/src/views/TopicsPage/index.vue

```diff
@@ -380,6 +380,8 @@
         topicMoreLoading: false,
         mobileSearchActive: false,
         currentSearchCardViewStyle: 'card',
+        closeCategoryModal: '',
+        sidePanelOverlayWidth: '',
```

Comment:
I'm wondering if these are values that should not exist? They're not referenced in any meaningful way (ie, if you initialize them here, there is no place where they are then updated to a meaningful value) -- so I think we either need to identify if these values need to be set on some condition or if we can just remove them altogether instead of just adding them into the scope of the component to suppress error messages

Reply:
My thought here is that some of the errors and values are leftover from refactoring that happened here over the course of some of the 0.15 patch release, so yes, agree with @nucleogenesis that making sure we have clarity about whether or not some more cleanup should happen is key. If you need some help sorting through that, @AllanOXDi, just let us know!